python/requirements-dev.txt

```diff
@@ -0,0 +1 @@
+idpb
```

Comment:
where is this being used?

Reply:
I was also wondering. I didn't find any reference to python/requirements-dev.txt, so I added it in `tox.ini` in my WIP PR: https://github.com/combust/mleap/pull/571/files#diff-b944be2c41cdc02da2ed52ea385d7677

`pyspark` would be a test-scope dependency, so I would add it to `requirements-dev.txt`.

Reply:
So requirements-dev.txt is generally not referenced in production code. It contains a development set of useful libraries and in this case ipdb is useful for debugging. But we could reference it in tox.ini since that's only called during testing

@juhoautio: I now added the file directly to the tox.ini and referenced the requirements.txt in requirement-dev.txt.